docs/sources/set-up/deploy.md

```diff
@@ -8,7 +8,120 @@ title: Deploy Grafana Alloy
 weight: 900
 ---
 
-{{< docs/shared source="alloy" lookup="/deploy-alloy.md" version="<ALLOY_VERSION>" >}}
+# Deploy {{% param "FULL_PRODUCT_NAME" %}}
+
+{{< param "PRODUCT_NAME" >}} is a flexible, vendor-neutral telemetry collector.
+This flexibility means that {{< param "PRODUCT_NAME" >}} doesn’t enforce a specific deployment topology but can work in multiple scenarios.
+
+This page lists common topologies used for {{% param "PRODUCT_NAME" %}} deployments, when to consider using each topology, issues you may run into, and scaling considerations.
+
+## As a centralized collection service
+
+Deploying {{< param "PRODUCT_NAME" >}} as a centralized service is recommended for collecting application telemetry.
+This topology allows you to use a smaller number of collectors to coordinate service discovery, collection, and remote writing.
+
+{{< figure src="/media/docs/alloy/collection-diagram-alloy.png" alt="Centralized collection with Alloy">}}
+
+Using this topology requires deploying {{< param "PRODUCT_NAME" >}} on separate infrastructure, and making sure that they can discover and reach these applications over the network.
+The main predictor for the size of an {{< param "PRODUCT_NAME" >}} deployment is the number of active metrics series it's scraping. A rule of thumb is approximately 10 KB of memory for each series.
+We recommend you start looking towards horizontal scaling around the 1 million active series mark.
+
+### Using Kubernetes StatefulSets
+
+Deploying {{< param "PRODUCT_NAME" >}} as a StatefulSet is the recommended option for metrics collection.
```

Comment:
I wonder if we need to start being more specific on otel vs prometheus. Maybe change to `metrics` to `Prometheus metrics`